src/main/scala/cromwell/binding/WdlImplicits.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.binding
+
+import cromwell.binding.types.{WdlIntegerType, WdlStringType}
+
+object WdlImplicits {
+  implicit class WdlString(val string: String) extends AnyVal {
```

Comment:
I think your comment about not loving it went away but that's what I mean and if you want to add implicit functions this is far more civilized.